server/channels/api4/export_test.go

```diff
@@ -41,11 +41,15 @@ func TestListExports(t *testing.T) {
 		exportDir := filepath.Join(dataDir, *th.App.Config().ExportSettings.Directory)
 		err := os.Mkdir(exportDir, 0700)
 		require.NoError(t, err)
-		defer os.RemoveAll(exportDir)
+		defer func() {
+			removeErr := os.RemoveAll(exportDir)
+			require.NoError(t, removeErr)
```

Comment:
You actually don't need to declare a new variable. You can reuse the existing one. That will also fix the linter issue: `Error: channels/api4/export_test.go:45:4: shadow: declaration of "err" shadows declaration at line 42 (govet)
`
```suggestion
			err = os.RemoveAll(exportDir)
			require.NoError(t, err)
```